tutorial/Lab1/README.md

```diff
@@ -0,0 +1,166 @@
+# Lab 1. I just want to deploy!
+
+Let's investigate how Helm can help us focus on other things by letting a chart do the work for us. We'll first deploy an application to a Kubernetes cluster by using `kubectl` and then show how we can offload the work to a chart by deploying the same app with Helm.
+
+The application is the [Guestbook App](https://github.com/IBM/guestbook), which is a a sample multi-tier web application.
+
+# Deploy the application using `kubectl`
+
+In this part of the lab, we will deploy the application using the Kubernetes client `kubectl`. We will use [Version 1](https://github.com/IBM/guestbook/tree/master/v1) of the app for deploying here. Clone the [Guestbook App](https://github.com/IBM/guestbook) repo to get the files: 
+```$ git clone https://github.com/IBM/guestbook.git``` .
+
+1. Use the configuration files in the cloned Git repository to deploy the containers and create services for them by using the following commands:
+
+   ```console
+   $ cd guestbook/v1
+
+   $ kubectl create -f redis-master-deployment.yaml
+   deployment.apps/redis-master created
+
+   $ kubectl create -f redis-master-service.yaml
+   service/redis-master created
+
+   $ kubectl create -f redis-slave-deployment.yaml
+   deployment.apps/redis-slave created
+
+   $ kubectl create -f redis-slave-service.yaml
+   service/redis-slave created
+
+   $ kubectl create -f guestbook-deployment.yaml
+   deployment.apps/guestbook-v1 created
+
+   $ kubectl create -f guestbook-service.yaml
+   service/guestbook created
+   ```
+   Refer to the [guestbook README](https://github.com/IBM/guestbook) for more details.
+ 
+2. View the guestbook:
+
+   Follow these [steps](https://github.com/IBM/guestbook/tree/master/v1#view-the-guestbook)
+
+Note: We were able to deploy the application by using `kubectl`. However, using `kubectl` here means that you already know the different resource files and the deployment of those resources in a particular order. It would be great if we could simplify this and just concentrate on the app deployment instead.
+
+# Deploy the application using Helm
+
+In this part of the lab, we will deploy the application by using Helm. We will use a release name of `guestbook-demo` to distinguish it from the previous deployment. The chart is available [here](https://github.com/IBM/helm101/tutorial/charts/guestbook).
+
```

Comment:
Are we creating a different chart here than what's available at https://github.com/IBM/helm101/tree/master/charts/guestbook